docs/system-admin-guide/integrations/nextcloud/README.md

```diff
@@ -70,7 +70,9 @@ In the configuration page that appears, you'll see a blank text field titled **O
 
 Click on the **Save** button. 
 
-> **Note:** If the OpenProject host cannot be added, you may check the [Troubleshooting](#troubleshooting) section at the bottom of this page
+> **Note:** If the OpenProject host cannot be added, you may check the [Troubleshooting](#troubleshooting) section at the bottom of this page.
+
+Please note that in order for the integration to work, the **Terms of Service** app on the Nextcloud side needs to be accepted. This is set to happen automatically during the initial set-up. If you see an error message indicating otherwise, please refer to the [Troubleshooting](#troubleshooting) section at the bottom of this page.
```

Comment:
```suggestion
Please note, when you use the **Terms of Service** app on the Nextcloud side, all terms also need to be accepted for the OpenProject user that gets created during the setup. This is set to happen automatically during the initial set-up. If you see an error message indicating otherwise or the integration does not behave as expected, please refer to the [Troubleshooting](#troubleshooting) section at the bottom of this page.
```